### PR TITLE
Update es-MX locale

### PR DESCRIPTION
**Pull request by jtapia**

### CODE REVIEW

File: config/locales/es-MX.yml [hunk 1]
```diff
@@ -286,6 +286,55 @@ es-MX:
...
+      - marzo
+      - abril
+      - mayo
+      - junio
+      - julio
+      - agosto
+      - septiembre
+      - octubre
+      - noviembre
+      - diciembre
```
Comment (tvdeyen) [inline config/locales/es-MX.yml]: Shouldn’t all these names be capitalized?

File: config/locales/es-MX.yml [hunk 2]
```diff
@@ -286,6 +286,55 @@ es-MX:
...
         short: "%b %-d '%y %-l:%M%P"
+  date:
+    abbr_day_names:
+      - lun
+      - mar
+      - mie
+      - jue
+      - vie
+      - sab
+      - dom
```
Comment (tvdeyen) [inline config/locales/es-MX.yml]: I think we should capitalize these

File: config/locales/es-MX.yml (hunk 1 above)
Comment (jtapia) [reply config/locales/es-MX.yml]: I was following the same format in other locales, but I made the change, thanks

File: config/locales/es-MX.yml (hunk 2 above)
Comment (jtapia) [reply config/locales/es-MX.yml]: done

File: config/locales/es-MX.yml [hunk 3]
```diff
@@ -286,6 +286,55 @@ es-MX:
       solidus:
         long: '%B %d, %Y %-l:%M %p'
         short: "%b %-d '%y %-l:%M%P"
+  date:
```
Comment (coorasse) [inline config/locales/es-MX.yml]: these are already provided by rails-i18n (https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/es-MX.yml#L10) and shouldn't be included here. sorry for the late reply 🙇  but could you please remove this `date` block?

Comment (jtapia) [reply config/locales/es-MX.yml]: done

**Pull request merged**